mws/resource_mws_log_delivery.go

```diff
@@ -59,10 +58,10 @@ func (a LogDeliveryAPI) Create(ldc LogDeliveryConfiguration) (string, error) {
 	return ld.LogDeliveryConfiguration.ConfigID, err
 }
 
-// Disable log delivery configuration - e.g. delete it
-func (a LogDeliveryAPI) Disable(accountID, configID string) error {
+// patch log delivery configuration - i.e. can only enable or disable it
+func (a LogDeliveryAPI) Patch(accountID, configID string, status string) error {
```

Comment:
does this mean we are not being backward compatible with it?

Reply:
this is an internal change, so terraform resource has no compatibility issues.